<launch>

  <arg name="config_file" default="$(find-pkg-share bloomxai_ros)/config/bloomxai_bag.yaml"/>

	<node pkg="bloomxai_ros" exec="bloomxai_server_node" name="bloomxai_server">

		<!--Config file -->
    <param from="$(var config_file)" />

		<!-- data source to integrate (PointCloud2) -->
		<remap from="/cloud_in" to="/camera_sensor/point_cloud" />

		<!-- Publish Height colormap on Resulting VoxelGrid -->
		<param name="use_height_map" value="true" />

	</node>
</launch>
